README.md

```diff
@@ -1,2 +1,66 @@
-# EventFinder_
+# EventFinder_Impulse
+
+## Impulse
+
+## Table of contents:
+- [Team Members](#Team-Members)
+- [Project Description](#Project-Description)
+- [User Story](#User-Story)
+- [Acceptance Criteria](#Acceptance-Criteria)
+- [Project Requirements](#Project-Requirements)
+
+
+## Team Members:
+* Brendan Moore
+* Hannelize Human (Project Manager)
+* Jose Gaspar
+* Michael Pappas
+<br>
+
+## Project Description:
+
 The aim of Project 1 is to collaboratively build a web application from scratch that allows the user to access and save a series of events based on their preferences. 
+<br>
+
+## User Story:
+```
+AS someone who enjoys going to events
+I WANT to have access to the different evets happening and be able to save my events
+SO THAT I can keep track of the events that I want to go to
+```
+
+## Acceptance Criteria:
+```
+GIVEN I am using a daily planner to access events across Sydney
+WHEN I open Impulse
+THEN I am presented with a [filter options]
+WHEN I enter my preferences
+THEN I am presented with upcoming event options
+WHEN I select an event I am interested in
+THEN I can add event to my calendar 
+```
+
+## Project Requirements:
+* Include two server-side API’s
+    * Ticketmaster  
+    * Google Calander
```

Comment:
There is a small typo . i think is Google Calendar

Reply:
Thanks for picking that up.